### ISSUE TITLE
Floating IP is not frees from the network leases

### ISSUE THREAD

**Issue by kvaps** [OpenNebula/one#1520]
# Bug Report
## Version of OpenNebula

- [ ] 5.2.2
- [ ] 5.4.0
- [ ] 5.4.1
- [ ] 5.4.2
- [ ] 5.4.3
- [X] Development build

## Component
- [X] Core & System
- [ ] Authorization (LDAP, x509 certs...)
- [ ] Command Line Interface (CLI)
- [ ] Contextualization
- [ ] Documentation
- [ ] KVM
- [ ] Networking
- [ ] Orchestration (OpenNebula Flow)
- [ ] Sunstone
- [ ] Storage
- [ ] Scheduler
- [ ] vCenter

### Expected Behavior
The Floating IP is not free after detaching or VR removing, if it was added after VR created. And can not be reused.

### Actual Behavior
The IP should be completly free after detaching or VR removing.

## How to reproduce
* Create new virtual router without interfaces.
* Attach new interfaces with "Floating IP" feature.
* Check your virtual router interfaces. *You will see that it have new interface, but without Floating IP defined, unlike all VMs from this VR will have it*
* Detach your interface or remove virtual router.
* Check your network leases, you will see that this IP is still served for `VR:<id>`.

**Comment by kvaps** [comment]
I think, it can be connected with #1505

**Comment by kvaps** [comment]
CC @juanmont

**Comment by kepi** [comment]
Is there any way how to free these leases after fix? I updated to 5.4.5 but there are still leases owned by vrouters I already deleted and I'm not able to delete them (or ARs).

**Comment by kepi** [comment]
Ok, I'm playing with this for couple hours now and I'm pretty sure this is not fixed. My nebula version is 5.4.5 build from git. Did my best to double check that I built correct version.

Every time I create new virtual router and assign floating IP to one of its NIC's, it remains leased even after I delete router. I tried to stop router instances before I deleted router, I tried to delete router with instances running etc, result is always the same.
